lib/amorail.rb

```diff
@@ -1,15 +1,15 @@
-require "amorail/version"
-require "amorail/config"
-require "amorail/client"
-require "amorail/exceptions"
-require "amorail/entity"
-require "amorail/custom_fields"
+require 'amorail/version'
+require 'amorail/config'
+require 'amorail/client'
+require 'amorail/exceptions'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.